linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/CoreJSLib.scala

```diff
@@ -1444,14 +1445,17 @@ private[emitter] object CoreJSLib {
                 genCallHelper(VarField.throwArrayIndexOutOfBoundsException, i))
           }
 
+          val getName = genArrayClassProperty(ArrayClassProperty.get)
+          val setName = genArrayClassProperty(ArrayClassProperty.set)
+
           List(
-              MethodDef(static = false, Ident("get"), paramList(i), None, {
+              MethodDef(static = false, getName, paramList(i), None, {
```

Comment:
Do the method defs here need original names?

Reply:
Hum, yes, they should have one.